.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "act-rules-repo"]
```

Comment:
I question the need to make this a submodule. Working with submodules adds complexity that I'm not sure we want (e.g. you now have to have the `--recursive` flag to clone the repo). Is there a specific reason why we couldn't npm install the repo on the master branch and use the needed file from a path into node-modules?

Reply:
I haven't quite figure out what I want with this yet. I'm not sure these tests should run on every PR, since the test suite changes, a new ACT test case could become a blocker to PRs. On the other hand, if I'm updating a rule that has ACT test cases, we should automatically test that our changes didn't break the ACT implementation.

We'll discuss on Monday. I agree with you that if we decide to always run these, NPM is probably easier than gitmodules.